gpii/node_modules/flowManager/configs/cloudBased.development.all.local.json

```diff
@@ -31,6 +34,9 @@
                 "url": "http://localhost:8078"
             }
         },
+        "authServerUrls": {
+            "preferencesPost": "http://localhost:8081/preferences?view=%view"
```

Comment:
This is ok for this pull request but we need to leave a note that this is a new dependency weak spot in the design - the author of this config has to be able to get this URL right, and the details of it depend on, and duplicate, other parts of the configuration of the system.